napari/_qt/dialogs/preferences_dialog.py

```diff
@@ -251,33 +251,32 @@ def build_page_dialog(self, schema, values):
         values : dict
             Dictionary of current values set in preferences.
         """
-        settings = get_settings()
         builder = WidgetBuilder()
         form = builder.create_form(schema, self.ui_schema)
 
-        # Disable widgets that loaded settings from environment variables
-        section = schema["section"]
-        form_layout = form.widget.layout()
-        for row in range(form.widget.layout().rowCount()):
-            widget = form_layout.itemAt(row, form_layout.FieldRole).widget()
-            name = widget._name
-            disable = bool(
-                settings._env_settings.get(section, {}).get(name, None)
-            )
-            widget.setDisabled(disable)
-            try:
-                widget.opacity.setOpacity(0.3 if disable else 1)
-            except AttributeError:
-                # some widgets may not have opacity (such as the QtPluginSorter)
-                pass
+        # # Disable widgets that loaded settings from environment variables
```

Comment:
Is this a temporal comment while tests are fixed?

Reply:
yeah... just need to bring back the `_env_settings` or rethink it's access